editor/editor_inspector.cpp

```diff
@@ -2860,12 +2830,12 @@ void EditorInspector::update_tree() {
 						// Hacky because this isn't necessarily always guaranteed.
 						doc_name = docs[docs.size() - 1].name;
 					}
-					if (script_name != StringName()) {
-						label = script_name;
+					if (script_name != String()) {
```

Comment:
If you're changing it to string, then this should use `is_empty()`. Same below.

Reply:
This needs to be reverted as well 🙃 (same below). And then we should be good to go!